app/helpers/devise_helper.rb

```diff
@@ -15,10 +15,25 @@
 module DeviseHelper
   def devise_error_messages!
```

Comment:
It looks like newer Devise deprecates this method in favor of a view partial which might handle everything you're adding here.  If this is the case, then you could delete this whole helper file.  See https://github.com/heartcombo/devise/blob/main/app/helpers/devise_helper.rb and https://github.com/heartcombo/devise/blob/main/app/views/devise/shared/_error_messages.html.erb.  If that works then it would probably be good to go ahead and follow the advice of the deprecation warning and removed references to `devise_error_messages!` in the views in favor of rendering the new partial directly.